### PR TITLE
Add opt-out docs

### PR DESCRIPTION
**Pull request by marinhero**
- Add section in the docs for the opt-out feature

Trello: https://trello.com/c/9swjrqAb/7-epic-ability-for-enduser-to-opt-out-of-survey-web-sms-mobile-email-is-ok

### CODE REVIEW

File: source/includes/_opt_out_js.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+```js
+window.wootricSettings = {
+  email:'nps@example.com',// TODO: Required to uniquely identify a user. Email is recommended but this can be any unique identifier.
+  created_at: 1234567890, // TODO: The current logged in user's sign-up date as a Unix timestamp.
+  account_token: 'NPS-xxxxxxx',
+  show_opt_out: true
+};
+```
+
+You can offer a way to opt-out from surveys from within the modal. To do so, you must enable the ```show_opt_out``` flag.
```
Comment (hachibu) [inline source/includes/_opt_out_js.md]: You can offer `users` a way to opt-out...

Comment (marinhero) [reply source/includes/_opt_out_js.md]: Perfect, thanks!

**Review by hachibu — APPROVED** [review]
:shipit:

**Comment by marinhero** [comment]
Closing this after Deepa requested to keep this feature hidden. Sorry for the trouble Ray!

**Pull request closed without merging**